app/javascript/channels/client_channel.js

```diff
@@ -0,0 +1,20 @@
+export function getChannelName(url) {
+  return { channel: "ClientChannel", id: url.match(/\/clients\/(\d*)\/messages/)[1] }
+};
+
+export const callback = {
+  connected() {
+    console.log("We're connected!")
+  },
+
+  disconnected() {
+    console.log("We're disconnected!")
+  },
+
+  received(data) {
+    let selector = data[0];
+    let newContent = data[1];
+
+    document.querySelector(selector).innerHTML += newContent;
```

Comment:
I wouldn't expect this to take an arbitrary selector. But totally understand this preliminary to get things working.